src/content/docs/dropins/user-account/functions.mdx

```diff
@@ -0,0 +1,578 @@
+---
+title: User account functions
+description: Learn about the API functions provided by the User Account dropin.
+---
+
+import OptionsTable from '@components/OptionsTable.astro';
+
```

Comment:
Consider adding a short lead-in sentence like [cart](https://experienceleague.adobe.com/developer/commerce/storefront/dropins/cart/cart-functions/) and [PDP](https://experienceleague.adobe.com/developer/commerce/storefront/dropins/product-details/pdp-functions/).